ocfweb/announcements/templates/announcements/2017-03-20-hiring.html

```diff
@@ -0,0 +1,121 @@
+{% extends 'base.html' %}
+{% load staticfiles %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-sm-8 ocf-content-block">
+            <h3></h3>
+            <p><em>March 20, 2017</em></p>
+            <hr />
+            <p>
+                The OCF is making a big transition to being completely
+                self-staffed next semester, and as part of that process, we are
```

Comment:
Well, technically everything will be self-staffed now, but yes, that would be more accurate.